### ISSUE TITLE
Put assets from Gem into `phaser-spaceshooter`

### ISSUE THREAD

**Issue by Gizmotronn** [Signal-K/client#6]
Also, the content in #5 needs to be moved into a separate folder/directory called "`final`" so that the existing work-in-progress status is not removed.

**Comment by Gizmotronn** [comment]
#4

**Comment by Gizmotronn** [comment]
Copy of the message I've done for Gemini Station team to explain the two scenes so far:

> So what I’ve come up with is two different scenes
The first is a space-invaders type scene that is your ship, and a bunch of other ships all moving around on a 2d plane (we can manipulate the script so that it tracks the movement of the ships in the actual game on the Phaser canvas). I’ve just used generic backgrounds & other assets for now, as I just realised today that I’m going to have to turn your assets into sprite sheets and I just wanted to focus on the code/functionality for now). 
The second image is an actual turn-based scene. I’ve divided the scene into two areas: the gameplay area and the context menu area. We can customise the styling of this using custom css (check out Arwes.dev for a cool sci-fi styling framework)…
In the combat scene, the player can see two of their ships (we can obviously change this number) - the ships they own that are currently in combat (e.g. “Dreadnought” & “Crusader”). The middle section of the context area is the actions they can take - e.g. Attack, Flee, Shield/Defend, Special Actions. The left-most section is showing the ships belonging to the enemy. I’m putting in some work to customise this so we can see the HP & other attributes.
Running the code right now takes you into those two scenes. 
In the turn-based combat scene, the player’s ships are linked to their account (and I’m assuming there would be animations/sprite sheets for their ships so when they move the ship, animations occur). I’ve linked in the player objects to animations, but the enemy ships are just able to be static images of the opponent’s ships that are in-combat, so I was able to use two of the assets you sent me to display how the enemy ships would look.
Right now I’m working on combining these two scenes so the player needs to move the ships around while participating in combat - the “space invaders-type” scene is basically the scaffolding for that.

<img width="493" alt="Screen Shot 2021-10-05 at 10 31 03 pm" src="https://user-images.githubusercontent.com/31812229/136049426-35ca6d0c-5993-413f-97f9-9dba60cccea1.png">
<img width="645" alt="Screen Shot 2021-10-05 at 10 52 44 pm" src="https://user-images.githubusercontent.com/31812229/136049449-1d85aeef-02f3-4cb6-a0fe-d4a8fbb0ec09.png">
